dat/HTMLTag-translations.ini

```diff
@@ -304,6 +304,42 @@ err_compat = "Эта версия HTML Tag требует Notepad++ 8.3 или 
 err_config = "Файл сущности не найден"
 err_config_msg = "Это должно быть в файле|или|в файле"
 
+[serbian]
+menu_0  = "Pronađi podudarni tag"
+menu_1  = "Izaberi podudarne tag-ove"
+menu_2  = "Izaberi tag i njegov sadržaj"
+menu_3  = "Izaberi samo sadržaj tag-a"
+menu_4  = "Enkodiraj entitete"
+menu_5  = "Enkodiraj entitete (uključujući prelome redova)"
+menu_6  = "Dekodiraj entitete"
+menu_7  = "Enkodiraj Unicode karaktere"
+menu_8  = "Dekodiraj Unicode karaktere"
+menu_9  = "Automatsko dekodiranje entiteta"
+menu_10 = "Automatsko dekodiranje Unicode karaktera"
+menu_11 = "O HTML Tag-u..."
+menu_12 = "Automatsko dopunjavanje HTML entiteta"
+err_compat = "Instalirana verzija HTML Tag-a zahteva Notepad++ 8.3 ili novije. Komande dodatka su onemogućene."
+err_config = "Nedostaje datoteka sa entitetima"
+err_config_msg = ""
```

Comment:
Original english label missing

Reply:
See [CONTRIBUTING](https://github.com/rdipardo/nppHTMLTag/blob/default/CONTRIBUTING.textile#new-menu-localizations) for an explanation.